### PR TITLE
Explicitly fail for python3 versions < 3.3.

### PR DESCRIPTION
**Pull request by craigcitro**
we just need to add a check in `setup.py`.

### CODE REVIEW

**Comment by craigcitro** [comment]
PTAL @dhermes if you have a sec?

**Comment by dhermes** [comment]
@nicholaschen @craigcitro 

We have an [explicit section][1] explaining why we only support 3.3+.

In reality, enterprise didn't ever (still hasn't) really touch Py3. By forcing 3.3+, it's actually encouraging the community to move forward. See our reasoning for more. (And Craig it may be worth adding a similar section here.)

[1]: https://github.com/GoogleCloudPlatform/gcloud-python/blob/87273a15d9be2b56c7d51a3bea0b5884c2313af0/README.rst#supported-python-versions

File: setup.py [hunk 1]
```diff
@@ -22,7 +22,10 @@
 import sys
 
 if sys.version_info <= (2, 6):
```
Comment (dhermes) [inline setup.py]: So you've never support Python 2.6? I am surprised by this but SGTM.

**Comment by dhermes** [comment]
LGTM

File: setup.py (hunk 1 above)
Comment (craigcitro) [inline setup.py]: oh, this actually does the right thing -- the version tuple for `2.6` will always have a third component.

that said, i agree it reads awkwardly, so i'm tweaking.

**Comment by craigcitro** [comment]
OK, added some notes, submitting.

thanks again @nicholaschen for asking -- i wouldn't have thought to add a note. and thanks @dhermes for giving me boilerplate to steal. `;)`

**Pull request merged**